backend/models/shiftSchema.ts

```diff
@@ -0,0 +1,13 @@
+import * as mongoose from 'mongoose';
+
+const ShiftSchema = new mongoose.Schema(
+  {
+    id: Number,
+    hours: Number,
+  },
```

Comment:
Ahhh, it looks like you may have been referencing the first draft portion of the google doc. I made some minor changes after I looked at it and remembered some things that needed to be changed. Apologies; please refer to the [notion](https://www.notion.so/Backend-Structure-f0ece22b0cca48dbb1e225dda3e51902) for the more recent version of our data structure. A Shift will hold the `ObjectId`s of its corresponding event and the volunteer.